packages/api/src/feature_flag/validators.ts

```diff
@@ -17,3 +20,32 @@ export function validateCreateFeatureFlagRequest(
     throw validation.error;
   }
 }
+
+export function validateUpdateFeatureFlagRequest(
+  data: unknown
+): asserts data is UpdateFeatureFlagRequest {
+  const validation = Joi.object()
+    .keys({
+      ...fieldsFromAdminAuthentication,
+      description: Joi.string(),
```

Comment:
`description` should be required here too, per the specification

Reply:
Ok, it seemed a bit odd not to be required on creation but required on update. I thought it's a documentation mistake.

Reply:
Oh, you're actually right, description can't be required because this endpoint needs to be able to set `description` to NULL by not sending it. Could you correct the error in the documentation so it matches the endpoint?